README.md

```diff
@@ -1,268 +1,143 @@
+
+## Deploying Field Controller
+forseti2 is dependent on external libraries, so there will be some OS-dependent instructions.
+
+### [Lightweight Communications and Marshalling (LCM)](https://lcm-proj.github.io/)
+1. Follow Online instructions on how to build it on your system.
+2. Once LCM is built, move `lcm.jar` into the forseti2 directory to take advantage of `lcm-spy.sh` packet introspection.
+
+### Installing for Ubuntu
+1. Install LCM as detailed above
+2. Install other packages:
+```bash
+sudo apt-get install python-pygame;
+sudo apt-get install arduino;
+sudo pip install pyfirmata;
+```
+
+### Installing for MacOS
+1. Install LCM as detailed above
+2. Install other packages:
+```bash
+sudo pip install flask;
+sudo brew install pygame;
+```
+
+## Nodes
+### timer.py
 Maintains match state (timer and robot states)
 
-Inputs:
-
-- Match/Init : Match
-  + resets match timer when a new match is initialized
-- Timer/Control: TimeControl
-  + receives commands for the match timer (start, pause, reset)
-- Robot[0-3]/Estop: Estop
-  + uses current emergency stop state and overrides all robot state if in estop mode
-- Robot[0-3]/Override: Override
-  + determines whether or not robot is in manual override mode (overrides timers for manual operation)
-- Robot[0-3]/RobotState: RobotState
-  + sets robot state using manual override only if in override mode
-
-Outputs:
-
-- Timer/Time: Time
-  + publishes (every .3 secs) time elapsed in current match and stage, as well as the current stage name
-- Robot[0-3]/RobotControl: RobotControl
-  + publishes (every .3 secs) the state of the robot (estop?, autonomous?, enabled?)
-
-### lcm_ws_bridge.py ###
-
-TODO:
-
-- Optimization/stress testing
-- Packet sniffer for testing
-
-Bridges lcm to websockets so javascript applications can access the network
-
-Inputs:
-
-- *: *
-  + receives whatever the websocket clients request
-
-Outputs:
-
-- *: *
-  + publishes whatever the websocket clients request
-
-### GameObjectTimer.py ###
-
+#### Inputs
+* Match/Init: Match
+  * Resets match timer when a new match is initialized
+* Timer/Control: TimeControl
+  * Receives commands for the match timer (start, pause, reset)
+* Robot[0-3]/Estop: Estop
+  * Uses current emergency stop state and overrides all robot state if in estop mode
+* Robot[0-3]/Override: Override
+  * Determines whether or not robot is in manual override mode (overrides timers for manual operation)
+* Robot[0-3]/RobotState: RobotState
+  * Sets robot state using manual override only if in override mode
+
+#### Outputs
+* Timer/Time: Time
+  * Publishes (every .3 secs) time elapsed in current match and stage, as well as the current stage name
+* Robot[0-3]/RobotControl: RobotControl
+  * Publishes (every .3 secs) the state of the robot (estop?, autonomous?, enabled?)
+
+### lcm_ws_bridge.py
+Bridges LCM to Websockets so JavaScript applications can access the network
+
+### GameObjectTimer.py
```

Comment:
This file was for last year's game, but not this year's. I think I actually deleted it from the repo as part of cutting down on legacy code.